modules/@apostrophecms/doc-type/index.js

```diff
@@ -371,18 +380,25 @@ module.exports = {
 
   methods(self) {
     return {
-      async updateCacheField(req, doc) {
-        const relatedDocsIds = self.getRelatedDocsIds(req, doc);
-
-        // - Remove current doc reference from docs that include it
-        // - Update these docs' cache field
-        await self.apos.doc.db.updateMany({
+      async deleteRelatedReverseId(doc, del = false) {
+        const locales = doc.aposLocale && del ? [
+          doc.aposLocale.replace(':draft', ':published'),
+          doc.aposLocale.replace(':published', ':draft')
+        ] : [ doc.aposLocale ];
```

Comment:
When deleting a document, we want to remove related reverse IDS to draft and published docs.

Reply:
Either this is missing parens so I can follow it better, or it always results in an array containing null in the "else" case...

Reply:
No the deleting option is only passed when we perform a delete draft.
During an update I kept the exact same behavior as before which was just passing `doc.aposLocale` as it is.
if I have `doc.aposLocal` AND the `deleting` option that is passed only when deleting draft or unlocalized doc, we want to remove related reverse ids from published and draft docs.
In the case of the update I wasn't totally sure so I kept the behavior since already working.